operator/builtin/input/tcp/tcp.go

```diff
@@ -15,6 +15,16 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	// InitialBufferSize is the initial size used for buffering
+	// TCP input
+	InitialBufferSize = 64*1024
```

Comment:
nit: Should we call this `minBufferSize`? Doesn't look like it needs to be exported.